app/views/shared/activities/_activity_summary.html.haml

```diff
@@ -45,3 +45,25 @@
       %dd.govuk-summary-list__actions
         - if policy(activity_presenter).update_linked_activity?
           = a11y_action_link(t("default.link.#{activity_presenter.call_to_action(:linked_activity)}"), activity_step_path(activity_presenter, :linked_activity), t("activerecord.attributes.activity.linked_activity"))
+
+- if activity_presenter.title.blank?
```

Comment:
What do you think about making this a more generic method, e.g. `activity_presenter.can_be_deleted?` and containing the deletion conditions in one place, to be more resilient to the possibility of deletion conditions changing in future?

Reply:
I like the idea of a method like that. By `containing the deletion conditions in one place` do you mean moving them out of the policy?

Reply:
Ah, no, in the policy it would be fine - I only meant that the condition shouldn't live in the view template.